src/shared/components/ChannelPreview/ChannelPreviewBase.tsx

```diff
@@ -1,37 +1,78 @@
 import React from 'react'
-import { AvatarContainer, Info, InnerContainer, OuterContainer } from './ChannelPreviewBase.style'
+import {
+  Anchor,
+  AvatarContainer,
+  Info,
+  InnerContainer,
+  NameHeader,
+  OuterContainer,
+  StyledAvatar,
+  VideoCount,
+  VideoCountContainer,
+} from './ChannelPreviewBase.style'
 import Placeholder from '../Placeholder'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+import { transitions } from '@/shared/theme'
 
 type ChannelPreviewBaseProps = {
-  avatarNode?: React.ReactNode
-  nameNode?: React.ReactNode
+  avatarUrl?: string
+  handle?: string
+  videoCount?: number
+  channelHref?: string
   className?: string
   animated?: boolean
```

Comment:
We have the correct condition now but I think `animated` prop isn't needed anymore?

Reply:
Right, forgot about the prop